scripts/utils.js

```diff
@@ -42,8 +43,23 @@ async function deployGatewayToken(gateway, tokenName, tokenSymbol, tokenDecimals
     await (await gateway.deployToken(params, commandId)).wait();
 }
 
+const getGasOptions = () => {
+    return network.config.blockGasLimit ? { gasLimit: network.config.blockGasLimit.toString() } : { gasLimit: 5e6 }; // defaults to 5M gas for revert tests to work correctly
+};
+
+const expectRevert = async (txFunc, contract, error) => {
+    if (network.config.skipRevertTests) {
+        await expect(txFunc(getGasOptions())).to.be.reverted;
+    } else {
+        await expect(txFunc(null)).to.be.revertedWithCustomError(contract, error);
+    }
+};
+
 module.exports = {
+    getChainId: async () => await network.provider.send('eth_chainId'),
```

Comment:
define above instead for consistency

Reply:
can also retrieve chain id from network.config.chainId instead to reduce RPC calls

Reply:
will change to `network.config.chainId`